app/services/health_checks/gov_uk_notify.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module HealthChecks
+  class GovUkNotify
+    URL = 'https://status.notifications.service.gov.uk'
+    COMPONENTS = ['API', 'Email sending'].freeze
+    OPERATIONAL = 'operational'
+    NAME = 'name'
+    STATUS = 'status'
+
+    def healthy?(force: false)
+      Rails.cache.fetch('health_checks_gov_uk_notify_healthy', expires_in: 30.seconds, force: force) do
```

Comment:
it seemed sensible to avoid swamping the govuk notify status page - as they might ban us. Its only cached for 30 seconds so should be pretty minimal delay